receiver/redisreceiver/README.md

```diff
@@ -0,0 +1,40 @@
+# Redis Receiver
```

Comment:
@tigrannajaryan There might be config options or metric names/labels that we want to change as we learn more when implementing additional monitors/receivers that would break compatibility. Should we mark this as beta/experimental somehow so we don't have to worry about making breaking changes yet?